src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -463,19 +519,35 @@ public CourseDetailsBundle getCourseSummary(String courseId) throws EntityDoesNo
         return getCourseSummary(cd);
     }
     
+    /**
+     * @param instructorAttributes
+     * @return the {@link CourseSummaryBundle course summary}, including
+     *         its feedback sessions using the given {@link InstructorAttributes}.
+     * @throws EntityDoesNotExistException
+     */
     public CourseSummaryBundle getCourseSummaryWithFeedbackSessionsForInstructor(
-            InstructorAttributes instructor) throws EntityDoesNotExistException {
-        CourseSummaryBundle courseSummary = getCourseSummaryWithoutStats(instructor.courseId);
-        courseSummary.feedbackSessions.addAll(feedbackSessionsLogic.getFeedbackSessionListForInstructor(instructor));
+            InstructorAttributes instructorAttributes) throws EntityDoesNotExistException {
```

Comment:
I missed this out earlier, but I feel that the changes should be related to this PR, i.e. adding the javadoc comments. Perhaps if you'd like to refactor/rename variables, create a separate issue for it and give your suggestions on what the variables should be renamed to in that issue?